src/issues/issue-2/wedding/post-wedding.js

```diff
@@ -0,0 +1,185 @@
+import React from "react"
+import Img from "gatsby-image"
+import { useStaticQuery, graphql } from "gatsby"
+
+import {
+  Wrapper,
+  TitleWrapper,
+  Title,
+  Authors,
+  Cover,
+  Article,
+  Row,
+  ArticleText,
+  ImageWrapper,
+  P,
+} from "./styled"
+import { findFileByName } from "../../../utils"
+
+const Wedding = () => {
+  const data = useStaticQuery(graphql`
+    query {
+      allFile(
+        filter: { relativeDirectory: { eq: "issue2-images/wedding-images" } }
+      ) {
+        edges {
+          node {
+            name
+            childImageSharp {
+              fixed {
+                ...GatsbyImageSharpFixed_withWebp_tracedSVG
+              }
+              fluid {
+                ...GatsbyImageSharpFluid_withWebp_tracedSVG
+              }
+            }
+          }
+        }
+      }
+    }
+  `)
+
+  const weddingCover = findFileByName(data, "wedding-cover")
```

Comment:
since we've been appending `img` to image variables in all other pages, let's stick to that convention here e.g, `weddingCoverImg`